lib/reviews.js

```diff
@@ -11,15 +11,13 @@ function reviews (opts) {
   return new Promise(function (resolve, reject) {
     validate(opts);
 
-    const sort = convertSort(opts.sort);
-
     const options = {
       method: 'POST',
       uri: 'https://play.google.com/store/getreviews',
       form: {
         pageNum: opts.page || 0,
         id: opts.appId || opts.id,
-        reviewSortOrder: sort,
+        reviewSortOrder: opts.sort,
```

Comment:
yeah, you are right
sorry, I notice this now and seems it's already done in `dev` branch
thanks :)